code/game/machinery/gulag_item_reclaimer.dm

```diff
@@ -9,6 +9,11 @@
 	var/list/stored_items = list()
 	var/obj/machinery/gulag_teleporter/linked_teleporter = null
 
+/obj/machinery/gulag_item_reclaimer/handle_atom_del(atom/deleting_atom)
+	for(var/person in stored_items)
```

Comment:
why is the var named person lol, we're storing items

Reply:
Storage is per person

Stored_items[Azerthene Severn]